digits/framework_helpers/caffe_helpers.py

```diff
@@ -0,0 +1,269 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+import caffe_pb2
+from google.protobuf import text_format
+
+# Constants
+CAFFE_SOLVER_FILE = 'solver.prototxt'
+CAFFE_ORIGINAL_FILE = 'original.prototxt'
+CAFFE_TRAIN_VAL_FILE = 'train_val.prototxt'
+CAFFE_SNAPSHOT_PREFIX = 'snapshot'
+CAFFE_DEPLOY_FILE = 'deploy.prototxt'
+CAFFE_WEIGHTS_FILE = 'model.caffemodel'
+
+class Error(Exception):
+    pass
+
+class CaffeSanityCheckError(Error):
+    """A sanity check failed"""
+    pass
+
+def filterLayersByState(network):
```

Comment:
@Lucaszw is just copying my code from https://github.com/NVIDIA/DIGITS/pull/628. I'm sure I've been inconsistent with this throughout the source tree. If we decide we want to fix it (and possibly add a linter to enforce it), we can do that in a later PR.